s2i/tensorflow-cm/Dockerfile

```diff
@@ -0,0 +1,2 @@
+FROM tensorflow/tensorflow:1.10.1-py3
+LABEL maintainer="Kubeflow"
```

Comment:
maintainer = "Kubeflow"?  use Caicloud or gaocegege?

Reply:
I think we could just remove the image since we do not use it now.